Jenkinsfile

```diff
@@ -132,5 +132,7 @@ pipeline {
     DATABASE_USER = 'backend'
     DATABASE_PASSWORD = credentials('database-password')
     APP_SECRET = credentials('app-secret')
-  }
+    For stage env: STAGE=sta
+    For prod env: STAGE=pro
+    }
```

Comment:
Is that correct?

Reply:
No, you need to put those in the docker commands, just like `DATABASE_HOST`.

Reply:
In fact, the environment section here should not have `DATABASE_HOST` at all.

Reply:
Disregard the last comment, it will be used for migrations, I created https://github.com/projekt-zespolowy-lab04-2020/backend/issues/32 for separating this

Reply:
So write here only STAGE='LOCAL' for example?

Reply:
Don't write anything here and add it here
https://github.com/projekt-zespolowy-lab04-2020/backend/blob/master/Jenkinsfile#L83
and here
https://github.com/projekt-zespolowy-lab04-2020/backend/blob/master/Jenkinsfile#L124